lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -27,10 +25,13 @@ define((require, exports, module) ->
                 rangeRight = new Range(0, 0, pos.right.row, pos.right.column + 1)
                 session.$bracketHighlightRight = session.addMarker(rangeRight, "ace_error-marker", "text")
         session.$highlightRange = pos
+        if pos.left && pos.right
+            if (pos.right.row - pos.left.row) > 70
```

Comment:
I have to disappoint you: my screen at this very moment accomodates 51 line. If I change the window size, it will accomodate less. If I hit F11it will accomodate more. 

https://ace.c9.io/api/editor.html#Editor.getFirstVisibleRow